api/src/main/resources/workbench.yaml

```diff
@@ -277,6 +277,44 @@ paths:
           schema:
             $ref: "#/definitions/ErrorResponse"
 
+  /v1/update-page-visits:
```

Comment:
Putting "update" in the name for a POST method feels a bit redundant, I think `POST /v1/page-visits` and `GET /v1/page-visits` make sense.

Reply:
+1, make use HTTP verbs in endpoints.